### ISSUE TITLE
Re: locked issue #240 - macOS 10.12.3 now out with bugfix from Apple!

### ISSUE THREAD

**Issue by SteveJobzniak** [codykrieger/gfxCardStatus#272]
Great news! I never thought we would see the day of Apple caring about their older hardware, but here it is!

http://forums.macrumors.com/threads/warning-bugs-in-sierra-which-affect-all-macbook-pro-2010-models.2000559/page-3#post-24237506

In case Codykrieger was too autistic to click the link before slagging me off in the previous ticket, the contents of the link are as follows:

> This is regarding the state of previous dual-GPU MacBook Pros again. There’s been a development on that front, so I’m writing to make you all aware.
> 
> macOS 10.12.3 contains some fixes to Apple’s frameworks!
> 
> 
> CoconutBattery:
> - Not fixed by 10.12.3.
> - Uses Core Animation, which is the cause of it.
> - When Core Animation was removed from the app (in a private build), the application no longer required the discrete GPU.
> 
> Siri:
> - Still uses discrete GPU in 10.12.3.
> - Because it uses Core Animation.
> 
> BetterTouchTool:
> - Fixed by 10.12.3.
> 
> **gfxCardStatus 2.3 (last official version):**
> - **Fixed by 10.12.3.**
> - **Previously, the only way to fix it was to re-compile it with newer frameworks.**
> - The recompiled version linked somewhere in this thread is still recommended, however, because it contains bugfixes not merged into the abandoned official version.
> 
> 
> The issue that Apple has fixed in 10.12.3 was that the mere existence of certain versions of certain frameworks (without the app even opening any OpenGL contexts) marked the app as “needs dGPU”. Now that’s no longer the case! WHAT A RELIEF!
> 
> 
> So the last *remaining* cause for dGPU is when applications use OpenGL *or* Core Animation. CA in turn opens an OpenGL context, which 100% always enables the discrete GPU. That has always been the case, and is a lot easier to avoid (most app makers avoid frivolous use of Core Animation).

In other words, the abandoned official version of @codykrieger's app no longer requires the discrete GPU in 10.12.3, and with no recompile necessary anymore. He can now close this ticket and resume letting it be abandoned without recompiling it or merging things like @steveschow 's Integrated GPU switching fix. 👍

**Comment by jingai** [comment]
Just because someone lost the time to work on something that matters to you doesn't mean they 'abandoned it'.

Maybe he just doesn't have time to work on it, yeah?

**Comment by codykrieger** [comment]
@SteveJobzniak: I read the link the first time you posted it. I locked the issue because:

1. You are treating GitHub Issues as a forum, which it is not—it is a place for reporting issues with gfxCardStatus.
2. You are incredibly rude, and are an enormous nuisance.

Also, please refrain from calling me autistic, as it's incredibly offensive to folks that actually are.
